### ISSUE TITLE
Option to change port

### ISSUE THREAD

**Issue by licaon-kter** [tananaev/rootless-logcat#7]

**Comment by tananaev** [comment]
What is the reason for changing port?

**Comment by licaon-kter** [comment]
Security, so this does not become a certain target for scans.

**Comment by tananaev** [comment]
Good point. I guess it a good idea to add some settings screen to the app.

**Comment by licaon-kter** [comment]
Also, if this can do it over the network, an actual address can be asked too, so you can see remote logs, like https://play.google.com/store/apps/details?id=com.cgutman.androidremotedebugger does it.

**Comment by tananaev** [comment]
Also agree. If we add setting page, we might as well add remote logcat option.

**Comment by hackel** [comment]
Wow, 9 years old!  I came here looking for this feature as well.  I want to be able to connect to remote devices over wireless adb, not just my phone.

This might be more complicated than simply specifying the IP address if it requires pairing with a code, however.
